src/backends/pyplot.jl

```diff
@@ -445,7 +445,7 @@ function py_add_series(plt::Plot{PyPlotBackend}, series::Series)
 
     # handle zcolor and get c/cmap
     needs_colorbar = hascolorbar(sp)
-    extrakw = if needs_colorbar
+    extrakw = if needs_colorbar || is_2tuple(sp[:clims])
```

Comment:
A user might also pass in e.g. `clims = 0:1` so I'd prefer `|| get(sp, :clims, nothing) != nothing`

Reply:
I agree, this would allow more input. However, `clims` defaults to `:auto` and this would have to be changed for all other backends too.

Reply:
good point